src/main/java/frc/robot/subsystems/gpm/Shooter.java

```diff
@@ -36,12 +51,34 @@ public void periodic() {
         topMotor.set(topPID.calculate(topSpeed) + feedforward.calculate(topPID.getSetpoint()));
         bottomMotor.set(bottomPID.calculate(bottomSpeed) + feedforward.calculate(bottomPID.getSetpoint()));
 
-        SmartDashboard.putNumber("top speed", ConversionUtils.shooterRPMToSpeed(topSpeed));
-        SmartDashboard.putNumber("bottom speed", ConversionUtils.shooterRPMToSpeed(bottomSpeed));
+        SmartDashboard.putNumber("top speed", shooterRPMToSpeed(topSpeed));
+        SmartDashboard.putNumber("bottom speed", shooterRPMToSpeed(bottomSpeed));
         SmartDashboard.putBoolean("at setpoint?", atSetpoint());
-        System.out.println(ConversionUtils.shooterRPMToSpeed(topSpeed) + "," + ConversionUtils.shooterRPMToSpeed(bottomSpeed));
+        System.out.println(shooterRPMToSpeed(topSpeed) + "," + shooterRPMToSpeed(bottomSpeed));
```

Comment:
Don't print periodically unless it's in a command specifically for testing/debugging or you plan to remove it before merging this PR.

Reply:
leftover code from testing, will remove